src/components/select/utils/with-filter.hoc.js

```diff
@@ -108,6 +106,7 @@ const withFilter = (WrappedComponent) => {
         multiColumn,
         noResultsMessage,
         tableHeader,
+        l.select,
```

Comment:
```suggestion
        l
```

I think this should compare the `locale` object itself incase the `noResultsForTerm` calls another function within the object.

Reply:
changed, thanks